src/Tile/_TileMedia.js

```diff
@@ -3,21 +3,31 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 const TileMedia = props => {
-    const { children, className, ...rest } = props;
+    const { children, className, productTile, backgroundImage, ...rest } = props;
 
     const tileMediaClasses = classnames(
-        'fd-tile__media',
+        {
+            'fd-tile__media': !productTile,
+            'fd-product-tile__media': productTile
+        },
         className
     );
 
-    return <div {...rest} className={tileMediaClasses}>{children}</div>;
+    return (
+        <div {...rest} className={tileMediaClasses}
+            style={{ backgroundImage: 'url(' + backgroundImage + ')' }}>{children}</div>);
```

Comment:
Should we only have this style block render if a background image exists? 
Otherwise we end up with this in the markup: 
`<div class="fd-tile__media" style="background-image: url("undefined");"><span class="fd-image--l fd-image--circle" style="background-image: url("https://placeimg.com/400/400/nature");"></span></div>`

See related comment on snapshots.